### ISSUE TITLE
Increased fecal protoporphyrin

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7550]
**Preferred term label:**
Increased fecal protoporphyrin

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormally high concentration of protoporphyrin (derivative of porphyrin with propionic acid groups) in feces.
PMID:8852667
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
176200 
PMID:8852667   4/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
